flight/Modules/OnScreenDisplay/inc/osd_utils.h

```diff
@@ -0,0 +1,177 @@
+/**
+ ******************************************************************************
+ * @addtogroup Tau Labs Modules
+ * @{
+ * @addtogroup OnScreenDisplay OSD Module
+ * @brief OSD Utility Functions
+ * @{
+ *
+ * @file       osd_utils.h
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2013-2015
+ * @author     The OpenPilot Team, http://www.openpilot.org Copyright (C) 2010-2014.
+ * @brief      OSD Utility Functions
+ * @see        The GNU Public License (GPL) Version 3
+ *
+ *****************************************************************************/
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ */
+
+#ifndef OSDUTILS_H
+#define OSDUTILS_H
+
+#include "openpilot.h"
+#include "fonts.h"
+#include "misc_math.h"
+
+// Size of an array (num items.)
+#define SIZEOF_ARRAY(x) (sizeof(x) / sizeof((x)[0]))
```

Comment:
There's a `#define NELEMENTS(x) (sizeof(x) / sizeof(*(x)))` defined in `pios.h`. (It's actually defined in many, many places and we might want to consider unifying that into our `shared/api` directory.)